user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Charts/Line_and_area_chart.md

```diff
@@ -70,6 +70,14 @@ To configure the component:
 
    - *Use percentage based values*: This option is only displayed if the component displays resource capacity information. If you select this option, the chart will display percentage values instead of absolute values.
 
+   - *Advanced* > *Hold Ctrl to zoom*: Available from DataMiner 10.4.0 [CU10]/10.5.1 onwards<!--RN 41387-->. Determines whether pressing the Ctrl key is required to zoom in or out.
```

Comment:
@WoutMH, I've adjusted the spelling to "Ctrl" here to be in line with the [Microsoft Style Guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/keys-keyboard-shortcuts#key-names); however, in the UI this is written as "ctrl". Is this something that could be updated in the UI?

Reply:
We can update this in all places, except for on the Maps component overlay. That message comes from the GoogleMaps API and cannot be changed. 
![image](https://github.com/user-attachments/assets/ba922868-3b36-4687-afb5-7a7b538d2299)